CMakeLists.txt

```diff
@@ -1,8 +1,15 @@
 # Basic parameters; check that these match your project / environment
 cmake_minimum_required(VERSION 3.8)
 
+# has to be before project
+if(PICO_SDK_PATH)
+    include(${32BLIT_DIR}/32blit-pico/pico_sdk_import.cmake)
+    include(${32BLIT_DIR}/32blit-pico/pico_extras_import.cmake)
+endif()
```

Comment:
There's a wrapper now for this: `include(${32BLIT_DIR}/32blit-pico/sdk_import.cmake)`. Also handles all the other methods of importing the SDK (environment, clone from git... (if that wasn't broken in -extras))